### PR TITLE
PETKA: Add support for compressed Red Comrades 1-2

### PR DESCRIPTION
**Pull request by tag2015**
This PR adds support for the compressed (not installed) versions of Red Comrades 1-2
Also changes the game titles to the correct Cyrillic title, since these games are Russian-only
and no translations are available it makes sense to keep the original titles in the launcher.

Fixes TRAC [11985](https://bugs.scummvm.org/ticket/11985)

### CODE REVIEW

File: engines/petka/detection.cpp [hunk 1]
```diff
@@ -26,8 +26,8 @@
 #include "petka/petka.h"
 
 static const PlainGameDescriptor petkaGames[] = {
-	{"petka1", "Red Comrades 1: Save the Galaxy"},
-	{"petka2", "Red Comrades 2: For the Great Justice"},
+	{"petka1", "Петька и Василий Иванович спасают галактику"},
```
Comment (sev-) [inline engines/petka/detection.cpp:29]: Umm, please revert the title changes. The titles are taken from the official English re-releases on Android.

File: engines/petka/detection_tables.h [hunk 2]
```diff
@@ -51,6 +51,19 @@ static const ADGameDescription gameDescriptions[] = {
...
+	{
+		"petka1",
+		"Compressed",
+		AD_ENTRY2s("BGS1.STR",  "99832accda859e2e1daeaae0a8561aeb", 20280804,
+				   "data1.cab", "7e73a644d8b15d2fd3781de5edce0c18", 228838940),
+		Common::RU_RUS,
+		Common::kPlatformWindows,
+		ADGF_DROPPLATFORM | ADGF_DROPLANGUAGE | GF_COMPRESSED,
+		GUIO1(GUIO_NOMIDI)
+
```
Comment (sev-) [inline engines/petka/detection_tables.h:64]: please remove this empty line

File: engines/petka/petka.cpp [hunk 3]
```diff
@@ -75,6 +76,12 @@ Common::Error PetkaEngine::run() {
 	initGraphics(640, 480, &format);
 	syncSoundSettings();
 
+	if(_desc->flags & GF_COMPRESSED) {
```
Comment (sev-) [inline engines/petka/petka.cpp:79]: bad formatting

**Review by sev- — CHANGES_REQUESTED** [review]
Nice that you were working on it. I left a couple of notes.

File: engines/petka/detection.cpp (hunk 1 above)
Comment (tag2015) [reply engines/petka/detection.cpp]: All right, done

File: engines/petka/detection_tables.h (hunk 2 above)
Comment (tag2015) [reply engines/petka/detection_tables.h]: Done, also removed empty lines for the other entries

File: engines/petka/petka.cpp (hunk 3 above)
Comment (tag2015) [reply engines/petka/petka.cpp]: fixed

**Comment by sev-** [comment]
Thank you!

**Pull request merged**